Docs/Data Sharing Framework/Creating Custom Components.md

```diff
@@ -0,0 +1,55 @@
+Creating Custom Components
+==========================
+
+Create the Component
+--------------------
+
+1. Create a LabVIEW project for your component(s).
+
+**Note**: It's recommended that you create two projects: Development and Build. Because this framework uses classes, they won't be editable when loaded under multiple target contexts. Use the Build project to include all your supported target types, which will lock the classes, and the Development project with just 'My Computer' to allow the classes to be edited.
```

Comment:
> **Note**: It's recommended that you create two projects: Development and Build. 

**Note**: It is recommended that you create separate projects for Development and Build. 

We currently use more than two projects, so I prefer a little more ambiguity rather than providing very specific guidance that we don't follow.